django_project/base/models/project.py

```diff
@@ -104,8 +104,9 @@ class Project(models.Model):
 
     image_file = models.ImageField(
         help_text=_('A logo image for this project. '
-                    'Most browsers support dragging the image directly on to '
-                    'the "Choose File" button above.'),
+                    'Most browsers support dragging the image '
+                    'directly on to the "Choose File" button above. '
+                    'The ideal size for your image is 512 x 512 pixels.'),
```

Comment:
It was a long road to get here, and one last step is to obey the 'pull left' rule. Please see:

http://kartoza.com/en/coding-standards/.  (search for pull left on that page)....